src/EnergyPlus/HVACSizingSimulationManager.cc

```diff
@@ -299,9 +299,10 @@ void ManageHVACSizingSimulation(EnergyPlusData &state, bool &ErrorsFound)
 
             while ((state.dataGlobal->DayOfSim < state.dataGlobal->NumOfDayInEnvrn) || (state.dataGlobal->WarmupFlag)) { // Begin day loop ...
 
-                if (ReportDuringHVACSizingSimulation) {
+                // Let's always do a transaction, except we'll roll it back if need be
+                // if (ReportDuringHVACSizingSimulation) {
                     if (sqlite) sqlite->sqliteBegin(); // setup for one transaction per day
-                }
+                // }
```

Comment:
Start a transaction, otherwise the next commit happening somewhere else will save the results anyways